pkg/apis/build/v1alpha1/build_test.go

```diff
@@ -0,0 +1,47 @@
+package v1alpha1_test
+
+import (
+	"testing"
+
+	"github.com/pivotal/kpack/pkg/apis/build/v1alpha1"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestRebaseable(t *testing.T) {
```

Comment:
Any chance this can be split into 3 spec tests? I understand this does the same thing but it would be nice for consistency